Code/Legacy/CrySystem/CMakeLists.txt

```diff
@@ -33,6 +33,8 @@ ly_add_target(
             Legacy::CrySystem.XMLBinary
             Legacy::RemoteConsoleCore
             AZ::AzFramework
+        PUBLIC
+            Gem::AudioSystem.API
```

Comment:
This is fine ... for now.
It is better than the Gem using CrySystem as a dependency.

Reply:
Yes, the idea is to further remove calls to audio system in CrySystem later.